apps/src/templates/sectionProgressV2/OpenAllStudentProgressButton.jsx

```diff
@@ -0,0 +1,149 @@
+import throttle from 'lodash/debounce';
+import PropTypes from 'prop-types';
+import React, {useState, useEffect, useRef} from 'react';
+import {connect} from 'react-redux';
+
+import PopUpMenu from '@cdo/apps/lib/ui/PopUpMenu';
```

Comment:
Could you explain why we are using PopUpMenu instead of SimpleDropdown or a custom component like SimpleDropdown that uses `<select>`?

Reply:
I started with `PopUpMenu` because it was what I was most familiar with... I wasn't aware of `SimpleDropdown` until later - now I know to start with storybook to look for components first.  That being said, according to this [slack thread](https://codedotorg.slack.com/archives/C06ERUABG01/p1718729235946379?thread_ts=1718664570.407079&cid=C06ERUABG01), it sounds like that won't meet our needs either.  Let me check in with Denys on this a bit more directly though too.